Mathlib/Tactic/FunProp/Theorems.lean

```diff
@@ -215,7 +215,7 @@ def getTheoremsForFunction (funName : Name) (funPropName : Name) :
 /-- General theorem about a function property used for transition and morphism theorems -/
 structure GeneralTheorem where
   /-- function property name -/
-  funPropName   : Name
+  funPropName : Name
   /-- theorem name -/
   thmName     : Name
```

Comment:
```suggestion
  thmName : Name
```
This and the fields below are currently not linted, but I still think that they should be: if you want to adapt this already, feel free to do so!  :smile: